src/sw.test.js

```diff
@@ -198,7 +198,7 @@ describe('Service Worker', () => {
 
     const cacheableAssets = [
       // Fonts
-      'https://ws-downloads.files.bbci.co.uk/fonts/ReithQalam/v1.210/bold.woff2',
+      'https://static.files.bbci.co.uk/fonts/ReithQalam/v1.310/bold.woff2',
```

Comment:
This file does not exist 😬

Reply:
Let's cache both of these files - check if we also need to add these links to the public/sw.js file
```suggestion
      'https://static.files.bbci.co.uk/fonts/reith-qalam/1.310/BBCReithQalam_W_Rg.woff',
      'https://static.files.bbci.co.uk/fonts/reith-qalam/1.310/BBCReithQalam_W_Bd.woff ',
```